sync/class.jetpack-sync-queue.php

```diff
@@ -54,6 +54,8 @@ function add( $item ) {
 			) );
 			$added      = ( 0 !== $rows_added );
 		}
+
+		do_action( 'jpsq_item_added' );
```

Comment:
We need documentation for this action. And perhaps it should be name `jetpack_sync_queue_item_added`.